app/controllers/sessions_controller.rb

```diff
@@ -8,7 +8,10 @@ def new
   def create
     user = User.find_by(username: params[:session][:username])
     current_user = user
-    if user && user.authenticate(params[:session][:password])
+    if user.admin?
+      flash[:welcomeadmin] = "Welcome, #{user.username.capitalize}!"
+      redirect_to '/admin/inventory'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.